RULES.md

```diff
@@ -135,6 +135,11 @@ A Route color and a Route text color should be contrasting. Minimum Contrast Rat
 
 ### E027 - Missing route short name and long name
 
+At least one of short name or long name should be provided - both can't be blank or missing.
```

Comment:
I would always mention the field name in the title or description of the rule, as it makes it easier to do Ctrl-F to find all rules related to a field.

```suggestion
At least one of `routes.route_short_name` or `routes.route_long_name` should be provided - both can't be blank or missing.
```